warrior/Framework/ClassUtils/rest_utils_class.py

```diff
@@ -80,7 +80,10 @@ def get(self, url, expected_response=None, params=None, auth=None, **kwargs):
         """
         pNote("Perform a http get", "info")
         try:
-            response = self.req.get(url, params=params, auth=auth, **kwargs)
+            if auth == (False, False,):
```

Comment:
You need to handle the same in post and put scenario's also.

Reply:
Handled all other scenario's and committed .